fxcore/assetlist.json

```diff
@@ -29,6 +29,36 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/fxcore/images/fx.svg"
         }
       ]
+    },
+    {
+      "description": "The cross chain token of the Function X",
+      "denom_units": [
+        {
+          "denom": "eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
+          "exponent": 0,
+          "aliases": [ ]
+        },
+        {
+          "denom": "PUNDIX",
+          "exponent": 18,
+          "aliases": [ ]
+        }
+      ],
+      "base": "eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
+      "display": "eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
```

Comment:
This `display` property is used to determine how many decimal places to use when representing $PUNDIX on various interfaces, and so should instead refer to 'PUNDIX' (the denom with exponent 18).